bcda/auth/provider.go

```diff
@@ -17,3 +21,13 @@ type Provider interface {
 	ValidateJWT(tokenString string) error
 	DecodeJWT(tokenString string) (jwt.Token, error)
 }
+
+func GetAuthProvider() Provider {
```

Comment:
With this being in `auth` now, could we call it `GetProvider()`?

Reply:
yep!